bridge/src/worker.rs

```diff
@@ -24,13 +23,6 @@ pub enum WorkerError {
     ChannelClosed(),
 }
 
-enum Request {
-    PollActivityTask {
-        core_worker: Arc<temporal_sdk_core::Worker>,
-        callback: WorkerCallback,
-    },
-}
-
 pub enum Response {
```

Comment:
Far as I can tell, the only variants `Response` needs are `Shutdown` (currently `Empty`), `Callback`. The `ActivityTask` and `Error` variants right now just pass bytes into the callback, but, the callback closure can simply be constructed with the bytes already moved into it.

Then receiving the message in the runtime becomes super simple. See my comment there.